.github/workflows/deploy-staging.yml

```
@@ -38,11 +70,14 @@ jobs:
 
   deploy-sanity-suite:
     name: Deploy sanity suite
-    needs: deploy
+    needs: [get-deploy-inputs]
     uses: ./.github/workflows/deploy-sanity-suite.yml
     with:
       environment: 'staging'
       use_pr_head_sha: 'true'
```

actionlint (1.7.7)

input "use_pr_head_sha" is typed as string by reusable workflow "./.github/workflows/deploy-sanity-suite.yml". bool value cannot be assigned

(expression)